[DOCS-5447] Bryce/add azure logging guide

buraizu opened the pull request:
<!-- *Note: Please remember to review the Datadog Documentation [Contribution Guidelines](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md) if you have not yet done so.* -->

### What does this PR do?
Moves the Azure logging instructions from the main Azure doc into a separate guide in the logs folder

### Motivation
DOCS-5413

<!-- ### Preview -->
<!-- Assuming you are a Datadog employee and named your branch `<yourname>/<description>`, a preview build will run and links to the preview output will be auto-generated and posted in the PR comments. The links will 404 until the preview build is finished running -->

### Additional Notes
<!-- Anything else we should know when reviewing?-->

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

maycmlee commented on content/en/logs/guide/azure-logging-guide.md:
```diff
@@ -0,0 +1,325 @@
+
+Once you have an App Registration configured, you can [create a log archive][62] that writes to Azure Blob Storage. 
+
+**Note**: If your storage bucket is in a subscription being monitored through the Azure Native integration, a warning is displayed in the Azure Integration Tile about the App Registration being redundant. You can ignore this warning.
```
```suggestion
**Note**: If your storage bucket is in a subscription being monitored through the Azure Native integration, a warning is displayed in the Azure Integration tile about the App Registration being redundant. You can ignore this warning.
```

maycmlee commented on content/en/logs/guide/azure-logging-guide.md:
```diff
@@ -0,0 +1,325 @@
+{{% /tab %}}
+{{< /tabs >}}
+
+## Log Archiving
```
```suggestion
## Log archiving
```

maycmlee commented on content/en/logs/guide/azure-logging-guide.md:
```diff
@@ -0,0 +1,325 @@
+To send logs from Azure to Datadog, follow this general process:
+
+1. Create an [Azure Event Hub][1].
+2. Setup the Datadog-Azure [function with an Event hub trigger][2] to forward logs to Datadog.
```
```suggestion
2. Set up the Datadog-Azure [function with an Event hub trigger][2] to forward logs to Datadog.
```

maycmlee commented on content/en/logs/guide/azure-logging-guide.md:
```diff
@@ -0,0 +1,325 @@
+
+{{% tab "Automated Installation" %}}
+
+To get started, click the button below and fill in the form on Azure Portal. The Azure resources required to get activity logs streaming into your Datadog account will be deployed for you.
```
```suggestion
To get started, click the button below and fill in the form on Azure Portal. The Azure resources required to get activity logs streaming into your Datadog account are deployed for you.
```

buraizu replied:
Thanks for catching this!

maycmlee commented on content/en/logs/guide/azure-logging-guide.md:
```diff
@@ -0,0 +1,325 @@
+2. Setup the Datadog-Azure [function with an Event hub trigger][2] to forward logs to Datadog.
+3. Configure your Azure services to stream logs to the Event Hub by creating a [diagnostic setting][3].
+
+The instructions below walk through a basic, initial setup using the Azure Portal. All of these steps can be performed with the CLI, Powershell, or resource templates by referring to the Azure documentation.
```
```suggestion
The instructions below walk through a basic, initial setup using the Azure portal. All of these steps can be performed with the CLI, Powershell, or resource templates by referring to the Azure documentation.
```

maycmlee reviewed (commented):
Just a couple of small nits! Also adding a comment in the Jira card.

buraizu commented:
Thanks for the review @maycmlee. I've incorporated your feedback, and also split the US3 information into a separate guide (content hasn't been changed though).

maycmlee reviewed (approved):
LGTM!

claudiadadamo commented on content/en/logs/guide/azure-logging-guide.md:
```diff
@@ -0,0 +1,275 @@
+2. Set up the [Datadog-Azure Function](#create-a-new-azure-blob-storage-function) which forwards logs from your blob storage to Datadog.
+3. Configure your Azure App Services to [forward their logs to the Blob Storage][6].
+
+#### Create a new Azure Blob Storage function
```
IMO we should add a note that this is not the suggested option for log forwarding to make it clear that we want customers to use eventhubs instead

buraizu replied:
Thanks for clarifying that -- I added a note to this effect, but let me know if you have any feedback on it.

buraizu commented:
Received approval from Azure team via Slack.
